src/main/java/org/folio/circulation/resources/LoanCollectionResource.java

```diff
@@ -202,14 +204,14 @@ void getMany(RoutingContext routingContext) {
     final ServicePointRepository servicePointRepository = new ServicePointRepository(clients);
     final LoanRepresentation loanRepresentation = new LoanRepresentation();
     final UserRepository userRepository = new UserRepository(clients);
+    final LoanPolicyRepository loanPolicyRepository = new LoanPolicyRepository(clients);
 
     loanRepository.findBy(routingContext.request().query())
-      .thenCompose(multiLoanRecordsResult ->
-        multiLoanRecordsResult.after(servicePointRepository::findServicePointsForLoans))
-      .thenCompose(multiLoanRecordsResult ->
-        multiLoanRecordsResult.after(userRepository::findUsersForLoans))
-      .thenApply(multipleLoanRecordsResult -> multipleLoanRecordsResult.map(loans ->
-        loans.asJson(loanRepresentation::extendedLoan, "loans")))
+      .thenComposeAsync(multiLoanRecordsResult -> multiLoanRecordsResult.after(servicePointRepository::findServicePointsForLoans))
```

Comment:
These lines were split because they significantly exceed the 80 character limit.

Why have they been combined into long lines?

Reply:
Reverted